src/script/conversation/MessageRepository.ts

```diff
@@ -1173,17 +1173,38 @@ export class MessageRepository {
    *
    * @param conversation Conversation message belongs to
    * @param messageId ID of message
-   * @param skipConversationMessages Don't use message entity from conversation
-   * @param ensureUser Make sure message entity has a valid user
    * @returns Resolves with the message
    */
-  async getMessageInConversationById(
+  async getMessageInConversationById(conversation: Conversation, messageId: string): Promise<StoredContentMessage> {
+    const messageEntity = conversation.getMessage(messageId);
```

Comment:
I believe the default value of this method is **not** to look up the local state. This might be a regression here (since now it always lookup to local state)

Reply:
correct